x-pack/plugins/security_solution/public/management/components/context_menu_with_router_support/context_menu_item_nav_by_rotuer.tsx

```diff
@@ -34,7 +36,20 @@ export const ContextMenuItemNavByRouter = memo<ContextMenuItemNavByRouterProps>(
         {...otherMenuItemProps}
         onClick={navigateAppId ? handleOnClickViaNavigateToApp : onClick}
       >
-        {children}
+        {textTruncate ? (
+          <div
+            className="eui-textTruncate"
+            style={{ maxWidth: '200px;' }}
```

Comment:
same, can we use some theme.eui prop in a styled component?

Reply:
I'm re-checking this.
I think it's a bug and this might have been left over from when I was playing with different approaches. Regardless, this can't be hard coded because the menu panel could have a different `maxWidth` from this and it would just distort the UI.

Reply:
Confirmed. this is not suppose to be here. I will delete it. Thanks for catching it